### PR TITLE
test: re-use self.stat() for verifying file/dir permissions

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]
Nice!

File: test/check-application [hunk 1]
```diff
@@ -1343,7 +1343,7 @@ class TestFiles(testlib.MachineCase):
         b.click(".pf-v5-c-modal-box button.pf-m-primary")
         b.wait_not_present(".pf-v5-c-modal-box")
 
-        self.assertEqual(m.execute(f"ls -ld {test_dir}")[:10], "dr-x------")
+        self.assertEqual(self.stat("%A", f"l{test_dir}"), "dr-x------")
```
Comment (martinpitt) [inline test/check-application:1346]: Ah, the extra "l" here breaks the show!

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: test/check-application (hunk 1 above)
Comment (martinpitt) [reply test/check-application:1346]: i.e. just drop the f-string

**Review by martinpitt — DISMISSED** [review]
Still fails, but I think that's a known flake now?

File: test/check-application [hunk 2]
```diff
@@ -1343,7 +1343,7 @@ class TestFiles(testlib.MachineCase):
         b.click(".pf-v5-c-modal-box button.pf-m-primary")
         b.wait_not_present(".pf-v5-c-modal-box")
 
-        self.assertEqual(m.execute(f"ls -ld {test_dir}")[:10], "dr-x------")
+        self.assertEqual(self.stat("%A", f"{test_dir}"), "dr-x------")
```
[the remark this answers is not in the document]
Comment (jelly) [reply test/check-application:1346]: ruff! Why u no complain ;-)

**Review by martinpitt — APPROVED** [review]

**Pull request merged**